cells/lib/ops/mkStandardOCI.nix

```diff
@@ -41,7 +41,10 @@ in
     meta ? {},
   }: let
     # Link useful paths into the container.
-    runtimeEntryLink = "ln -s ${l.getExe operable.passthru.runtime} $out/bin/runtime";
+    runtimeEntryLink = ''
+      ln -s ${l.getExe operable.passthru.runtime} $out/bin/sh
```

Comment:
It's been awhile since I've visited this. Why are linking the runtime to /bin/sh?

Reply:
I could just link the `runtimeShell` directly, but the runtime has the nice property of setting the path to the runtime inputs, so if a program depends on `/bin/sh` and that invocation calls out to some other program that already exists in the `runtimeInputs` it would suceed. If we just linked directly to `nixpkgs.runtimeShell` it would fail since the PATH would basically be empty.

Reply:
Ah, that makes sense.